lib/content-services/src/lib/dialogs/category-selector.dialog.spec.ts

```diff
@@ -45,11 +52,14 @@ describe('Category selector dialog component', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [CoreTestingModule],
```

Comment:
Should be using the testing module to avoid code repetition

Reply:
:D it is labeled as 'do not review' but i'll answer here then we can talk , CoreTestingModule is importing CoreModule so using it it's the wrong way to go, IF we decide to remove that from there then we can put the Auth part and some other repetitive stuff there to avoid reimporting them until then using it is a bad idea as it will load tons of stuff slowing and increasing the size of everything (we did this check many years ago now )